src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -0,0 +1,282 @@
+                        icon={Expensicons.Gear}
+                        iconStyles={[styles.mr2]}
+                    />
+                </>
+            ) : (
+                <ButtonWithDropdownMenu<WorkspaceDistanceRatesBulkActionType>
+                    shouldAlwaysShowDropdownMenu
+                    pressOnEnter
+                    customText={`${selectedDistanceRates.length} ${translate('workspace.distanceRates.selected')}`}
+                    buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
+                    onPress={() => null}
+                    options={getBulkActionsButtonOptions()}
+                    buttonRef={dropdownButtonRef}
+                    style={[isSmallScreenWidth && styles.flexGrow1]}
+                    wrapperStyle={styles.w100}
+                />
+            )}
+        </View>
+    );
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    style={[styles.defaultModalContainer]}
+                    testID={PolicyDistanceRatesPage.displayName}
+                    shouldShowOfflineIndicatorInWideScreen
+                >
+                    <HeaderWithBackButton
+                        icon={Illustrations.CarIce}
+                        title={translate('workspace.common.distanceRates')}
+                        shouldShowBackButton={isSmallScreenWidth}
+                    >
+                        {!isSmallScreenWidth && headerButtons}
+                    </HeaderWithBackButton>
+                    {isSmallScreenWidth && <View style={[styles.ph5]}>{headerButtons}</View>}
+                    <View style={[styles.ph5, styles.pb5, styles.pt3]}>
+                        <Text style={[styles.textNormal, styles.colorMuted]}>{translate('workspace.distanceRates.centrallyManage')}</Text>
+                    </View>
+                    <SelectionList
+                        canSelectMultiple
+                        ListItem={TableListItem}
+                        onSelectAll={toggleAllRates}
+                        onCheckboxPress={toggleRate}
+                        sections={[{data: distanceRatesList, indexOffset: 0, isDisabled: false}]}
+                        onSelectRow={editRate}
+                        showScrollIndicator
+                        customListHeader={getCustomListHeader()}
+                        listHeaderWrapperStyle={[styles.ph9, styles.pv3, styles.pb5]}
+                    />
+                    <Modal
+                        type={CONST.MODAL.MODAL_TYPE.CENTERED_SMALL}
+                        isVisible={isWarningModalVisible}
+                        onClose={() => setIsWarningModalVisible(false)}
+                    >
+                        <View style={[styles.notSoFastPopoverWrapper]}>
```

Comment:
Why is there such a specific style here? `notSoFastPopoverWrapper` seems like it's ultra specific, even though this modal is a pretty generic looking component.

Reply:
We can reuse existing `ConfirmModal`,this is what it'll look like. wdyt?

![Screenshot 2024-03-08 at 12 23 34 at night](https://github.com/Expensify/App/assets/75031127/a3013649-07b1-4207-84f5-c6256455f059)

Reply:
That would be ideal, yup. Maybe we can change the copy slightly so we don't get an orphan either, cc @trjExpensify

Reply:
Swapped to `ConfrimModal` component and removed extra styles Ive added